### PR TITLE
Nb/docs/warnings fix

### PR DESCRIPTION
**Pull request by Nadav-Barak**

### CODE REVIEW

File: deepchecks/core/serialization/suite_result/html.py [hunk 1]
```diff
@@ -196,7 +196,8 @@ def prepare_summary(self, output_id: t.Optional[str] = None, **kwargs) -> str:
         extra_info = self.prepare_extra_info()
 
         suite_creation_example_link = (
-            'https://docs.deepchecks.com/en/stable/examples/guides/create_a_custom_suite.html'
+            'https://docs.deepchecks.com/en/stable/general/usage/customizations/'
```
Comment (shir22) [inline deepchecks/core/serialization/suite_result/html.py:199]: without the "/en/stable" in all links

Comment (shir22) [reply deepchecks/core/serialization/suite_result/html.py:199]: (also please search and see that there aren't any /en/stable links left. should be only /stable)

File: deepchecks/nlp/checks/data_integrity/property_label_correlation.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
 PLC = t.TypeVar('PLC', bound='PropertyLabelCorrelation')
 
-pps_url = 'https://docs.deepchecks.com/en/stable/checks_gallery/tabular/' \
+pps_url = 'https://docs.deepchecks.com/en/stable/tabular/checks_gallery/' \
```
Comment (shir22) [inline deepchecks/nlp/checks/data_integrity/property_label_correlation.py:30]: this link doesn't work.
Didn't check the others, please verify that they work

File: deepchecks/nlp/text_data.py [hunk 3]
```diff
@@ -85,7 +85,7 @@ class TextData:
         In order to calculate the default properties, use the `TextData.calculate_default_properties` function after
         the creation of the TextData object.
         For more on properties, see the `NLP Properties Guide
-        <https://docs.deepchecks.com/en/latest/nlp/nlp-properties.html>`_.
+        <https://docs.deepchecks.com/en//stable/nlp/usage_guides/nlp_properties.html>`_.
```
Comment (shir22) [inline deepchecks/nlp/text_data.py:88]: there is a double // here

File: docs/source/getting-started/installation.rst [hunk 4]
```diff
@@ -63,12 +63,32 @@ Simply run the following command in a notebook cell
     !{sys.executable} -m pip install deepchecks --quiet --upgrade # --user
 
 
+Deepchecks For NLP
+==================
+
+.. note:: 
+   Deepchecks' NLP subpackage is in **beta** release, and is available from PyPi,
```
Comment (shir22) [inline docs/source/getting-started/installation.rst:70]: needed new line here?

File: docs/source/getting-started/installation.rst [hunk 5]
```diff
@@ -63,12 +63,32 @@ Simply run the following command in a notebook cell
...
+   use at your own discretion. `Github Issues <https://github.com/deepchecks/deepchecks/issues>`_ are
```
Comment (shir22) [inline docs/source/getting-started/installation.rst:71]: two underscores after a url link so it'll be "anonymous"

File: docs/source/getting-started/installation.rst (hunk 4 above)
Comment (shir22) [inline docs/source/getting-started/installation.rst:70]: PyPI

File: docs/source/nlp/index.rst [hunk 6]
```diff
@@ -6,13 +6,14 @@ NLP
 
 Deepchecks for NLP is currently in alpha stage.
 Visit the :ref:`installation_guide` page to set up your environment and than head over to the
```
Comment (shir22) [inline docs/source/nlp/index.rst:8]: "than" -> then in all of them

File: deepchecks/vision/suites/default_suites.py [hunk 7]
```diff
@@ -38,17 +38,17 @@ def train_test_validation(label_properties: List[Dict[str, Any]] = None, image_p
 
            * - Check Example
```
Comment (shir22) [inline deepchecks/vision/suites/default_suites.py:39]: do these fixes work?

**Review by shir22 — COMMENTED** [review]
I think most links still need updating. Please verify that they work (and/or that they fail build?)
Also - build docs? (and attach)

File: deepchecks/nlp/checks/data_integrity/property_label_correlation.py (hunk 2 above)
Comment (Nadav-Barak) [reply deepchecks/nlp/checks/data_integrity/property_label_correlation.py:30]: They will work, after we will merge

File: docs/source/getting-started/installation.rst (hunk 4 above)
Comment (Nadav-Barak) [reply docs/source/getting-started/installation.rst:70]: yea it fixed a warning

**Review by ItayGabbay — APPROVED** [review]

**Pull request merged**